src/Stratis.Bitcoin.Features.RPC/RpcSettings.cs

```diff
@@ -64,12 +80,18 @@ private void LoadSettingsFromConfig(NodeSettings nodeSettings)
             var config = nodeSettings.ConfigReader;
 
             this.Server = config.GetOrDefault<bool>("server", false);
+            this.logger.LogDebug("Server set to {0}.", this.Server);
+
             this.RPCPort = config.GetOrDefault<int>("rpcport", nodeSettings.Network.RPCPort);
+            this.logger.LogDebug("Port set to {0}.", this.RPCPort);
 
             if (this.Server)
             {
                 this.RpcUser = config.GetOrDefault<string>("rpcuser", null);
+                this.logger.LogDebug("RpcUser set to '{0}'.", this.RpcUser);
+
                 this.RpcPassword = config.GetOrDefault<string>("rpcpassword", null);
+                this.logger.LogDebug("RpcPassword set to '{0}'.", (this.RpcPassword == null) ? "" : "******");
```

Comment:
Please remove this as well. Logging "" is a security leak.

Reply:
There are other very quick and straightforward ways to determine if the password is "". This is not introducing any security leak that does not already inherently exist when using empty passwords. However any logging related to passwords have already been removed in the follow-up PR which comes after this one.